### PR TITLE
fixed simulator flag

### PR DESCRIPTION
**Pull request by StefanJakob**
simulator flag was public => set to private 
fixed code using public field

### CODE REVIEW

File: msl_base/src/Base.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ Base::Base(string roleSetName, string masterPlanName, string roleSetDir, bool si
     wm = MSLWorldModel::get();
     if (sim)
```
Comment (MasterMarci) [inline msl_base/src/Base.cpp]: can this be combined with the other if a few lines above?

File: msl_worldmodel/include/MSLWorldModel.h [hunk 2]
```diff
@@ -124,9 +125,9 @@ class MSLWorldModel
     msl_actuator_msgs::RawOdometryInfoPtr rawOdometry;
 
     supplementary::EventTrigger visionTrigger;
-    InfoTime timeLastSimMsgReceived;
 
   private:
+    InfoTime timeLastSimMsgReceived;
```
Comment (MasterMarci) [inline msl_worldmodel/include/MSLWorldModel.h]: Shouldn't this be a bool?

**Comment by MasterMarci** [comment]
@kiddikai Would you check the pull request pls?

**Comment by kiddikai** [comment]
void MSLWorldModel::setIsUsingSimulator()

This is counter-intuitive, I suggest passing a boolean parameter and setting the flag accordingly like you would expect from a usual setter method.

**Comment by BriseLiese** [comment]
WM init fixed. Has been tested.

**Pull request merged**